File: app/src/main/java/com/bijoysingh/quicknote/activities/CreateAdvancedListActivity.java
package com.bijoysingh.quicknote.activities;

import com.bijoysingh.quicknote.formats.FormatType;

public class CreateAdvancedListActivity extends CreateOrEditAdvancedNoteActivity {

  protected void addDefaultItem() {
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED);
  }

}

File: app/src/main/java/com/bijoysingh/quicknote/activities/CreateAdvancedListActivity.kt
package com.bijoysingh.quicknote.activities

import com.maubis.scarlet.base.format.FormatType

class CreateAdvancedListActivity : CreateOrEditAdvancedNoteActivity() {

  override fun addDefaultItem() {
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
    addEmptyItem(FormatType.CHECKLIST_UNCHECKED)
  }
}
